js/solstice.modal.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+(function($, document) {
+  $('#eclipsefdn-modal').on('show.bs.modal', function (event) {
```

Comment:
I think eclipsefdn-modal is too generic as this if for a session abstract modal.

How about we name this eclipsefdn-modal-event-session?